isis/tests/FunctionalTestsCsminit.cpp

```diff
@@ -318,7 +317,7 @@ TEST_F(DefaultCube, CSMinitSpiceCleanup) {
 TEST_F(DefaultCube, CSMinitSpiceNoCleanup) {
   // Create an ISD that will result in no successful models
   json isd;
-  isd["name"] = "failing_isd";
+  isd["model_name"] = "failing_isd";
```

Comment:
The model_name doesn't need to be in the isd at all anymore

Reply:
Good point! Thanks for pointing this out.